server/sonar-server/src/main/java/org/sonar/server/user/UserSession.java

```diff
@@ -35,6 +35,10 @@
   @CheckForNull
   Integer getUserId();
 
+  /**
+   * @deprecated does not support organizations as group names are not unique
+   */
+  @Deprecated
   Set<String> getUserGroups();
```

Comment:
I don't see any replacement provided in the PR.
This might be caused by the main consumer of this method is the issues domain (and the ES index) and the change to fix it is quite big and should be made in a dedicated PR, right?